### ISSUE TITLE
"ERROR: Message 'implementationClass' not understood.": when opening schelp file from Help Browser

### ISSUE THREAD

**Issue by prko** [scztt/vscode-supercollider#25]
Hello!

When clicking the Help file source in the last lines of a Help document in Help Browser,
<img width="1209" alt="Screenshot 2023-11-01 at 16 21 15" src="https://github.com/scztt/vscode-supercollider/assets/416281/8b16762b-b17c-4309-bcd4-f62dce1c813d">
the following error occurs (This error does not occur if you do the same thing in the help document, which can be accessed via "SuperCollider: Search Help" because the link does not work):
```
ERROR: Message 'implementationClass' not understood.
RECEIVER:
class Document (0x1410c4580) {
  instance variables [19]
    name : Symbol 'Document'
    nextclass : instance of Meta_DocumentFile (0x149261b40, size=19, set=5)
    superclass : Symbol 'Object'
    subclasses : instance of Array (0x179798f00, size=1, set=2)
    methods : instance of Array (0x1410c4700, size=27, set=5)
    instVarNames : instance of SymbolArray (0x1410c4980, size=9, set=3)
    classVarNames : instance of SymbolArray (0x1410c4c00, size=9, set=3)
    iprototype : instance of Array (0x1410c4a80, size=9, set=4)
    cprototype : instance of Array (0x1410c4d00, size=9, set=4)
    constNames : nil
    constValues : nil
    instanceFormat : Integer 0
    instanceFlags : Integer 0
    classIndex : Integer 1316
    classFlags : Integer 0
    maxSubclassIndex : Integer 1317
    filenameSymbol : Symbol '/Users/prko/Library/Application Support/SuperCollider/downloaded-quarks/LanguageServer/scide_vscode/LSPDocument.sc'
    charPos : Integer 1517
    classVarIndex : Integer 673
}
ARGS:
CALL STACK:
	DoesNotUnderstandError:reportError
		arg this = <instance of DoesNotUnderstandError>
	Nil:handleError
		arg this = nil
		arg error = <instance of DoesNotUnderstandError>
	Thread:handleError
		arg this = <instance of Thread>
		arg error = <instance of DoesNotUnderstandError>
	Object:throw
		arg this = <instance of DoesNotUnderstandError>
	Object:doesNotUnderstand
		arg this = <instance of Meta_Document>
		arg selector = 'implementationClass'
		arg args = [*0]
	String:openDocument
		arg this = "/Users/prko/Dropbox/prko/__m..."
		arg selectionStart = 0
		arg selectionLength = 0
		var ideClass = nil
	< FunctionDef in Method HelpBrowser:redirectTextFile >
		arg x = ".schelp"
		var path = "/Users/prko/Dropbox/prko/__m..."
	ArrayedCollection:do
		arg this = [*5]
		arg function = <instance of Function>
		var i = 3
	HelpBrowser:redirectTextFile
		arg this = <instance of HelpBrowser>
		arg url = "file:///Users/prko/Dropbox/p..."
		var plainTextExts = [*5]
	< FunctionDef in Method HelpBrowser:init >
		arg wv = <instance of WebView>
		arg url = "file:///Users/prko/Dropbox/p..."
		var redirected = nil
		var newPath = nil
		var oldPath = nil
	QObject:doFunction
		arg this = <instance of WebView>
		arg f = <instance of Function>
		arg args = [*3]
^^ ERROR: Message 'implementationClass' not understood.
RECEIVER: Document
```

**Comment by scztt** [comment]
Fixed by [b23b3a0a8387974543f36594d1a14ed4e10c0576](https://github.com/scztt/LanguageServer.quark/commit/b23b3a0a8387974543f36594d1a14ed4e10c0576).
You should be able to see this resolved if you run the "Update LanguageServer.quark" command in VSCode.

**Comment by prko** [comment]
Thanks!
